packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js

```diff
@@ -1,24 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './ProductListDropDown.style.css';
 
 const ProductListDropdown = ({ options, ...props }) => {
-  const optionList =
-    options.length > 0 &&
-    options.map((item) => {
-      return <option value={item}>{item}</option>;
-    });
+  const [value, setValue] = useState('');
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+  const categories = [
+    'Categories',
+    'Category 1',
+    'Category 2',
+    'Category 3',
+    'Category 4',
+    'Category 5',
+  ];
+  const sortBy = [
+    'Sort By Most Recent',
+    'Recent Collections',
+    'A-Z',
+    'Z-A',
+    'Price ↓',
+    'Price ↑',
+  ];
+  const allFilter = [
+    'All Filter',
+    'Price',
+    'Size',
+    'Color',
+    'Reviews',
+    'Brand',
+  ];
 
   return (
-    <div>
-      <select className="view-dropdown-select">{optionList}</select>
+    <div className="list-view">
+      <div>
+        <select
+          className="view-dropdown-select"
+          value={value}
+          onChange={handleChange}
+        >
+          {categories.map((optionCategories) => (
+            <option value={optionCategories}>{optionCategories}</option>
+          ))}
+        </select>
+      </div>
+      <div>
+        <select
+          className="view-dropdown-select"
+          value={value}
+          onChange={handleChange}
+        >
+          {sortBy.map((optionSortBy) => (
+            <option value={optionSortBy}>{optionSortBy}</option>
+          ))}
+        </select>
+      </div>
+      <div>
+        <select
+          className="view-dropdown-select"
+          value={value}
+          onChange={handleChange}
+        >
+          {allFilter.map((optionAllFilter) => (
+            <option value={optionAllFilter}>{optionAllFilter}</option>
+          ))}
+        </select>
+      </div>
```

Comment:
Those are 3 basically identical pieces of code.
Which in React means only one thing :) 
Make it into a component that gets what it needs through props ✅

Reply:
Thank you feedback. I have add separate components.